recipes/opencascade/all/conanfile.py

```diff
@@ -135,9 +135,10 @@ def source(self):
     def generate(self):
         tc = CMakeToolchain(self)
 
-        # Inject C++ standard from profile since we have removed hardcoded C++ standard from upstream build files
-        if not valid_min_cppstd(self, self._min_cppstd):
-            tc.variables["CMAKE_CXX_STANDARD"] = self._min_cppstd
+        if Version(self.version) >= "7.8.0":
+            cppstd = str(self.settings.compiler.cppstd).replace("gnu", "").upper()
+            tc.cache_variables["BUILD_CPP_STANDARD"] = f"C++{cppstd}"
```

Comment:
https://github.com/Open-Cascade-SAS/OCCT/blob/master/CMakeLists.txt#L25 this value actually is not enfornced because upstream has a , so it's not an actual fatal error